src/components/Card/MainCard.styled.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+/*pewnie  lepiej byłoby to dać w osobnych plikach ? ale swierdziłam, że skoro jest mało tego to dam w jednym dla mojej wygody ale nie wiem czy to jest ok?*/
```

Comment:
Może być ;) Jedynie chyba nazwę bym zmienił na `index.styled.js` :D ale to już od Ciebie zależy :)